spring-cloud-generator/scripts/generate.sh

```diff
@@ -57,8 +59,6 @@ echo "Library list path: ${LIBRARY_LIST_PATH}"
 
 cd ${SPRING_GENERATOR_DIR}
 
-echo "executing install_spring_generator"
-install_spring_generator
```

Comment:
Removed since locally installing spring-cloud-gcp from root (in action) already includes the generator module: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1957#discussion_r1252161199